packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -472,37 +532,84 @@ class InspectorRowContent extends StatelessWidget {
           child: Row(
             crossAxisAlignment: CrossAxisAlignment.center,
             textBaseline: TextBaseline.alphabetic,
-            children: [
-              node.showExpandCollapse
-                  ? InkWell(
-                      onTap: onToggle,
-                      child: RotationTransition(
-                        turns: expandAnimation,
-                        child: Icon(
-                          Icons.expand_more,
-                          size: 16.0,
-                        ),
-                      ))
-                  : const SizedBox(width: 16.0, height: 16.0),
-              SizedOverflowBox(
-                size: Size(controller.rowWidth, rowHeight),
-                alignment: Alignment.centerLeft,
-                child: DecoratedBox(
-                  decoration: BoxDecoration(
-                    color: backgroundColor,
-                  ),
-                  child: InkWell(
-                    onTap: () {
-                      controller.onSelectRow(row);
-                    },
-                    child: Container(
-                        height: rowHeight,
-                        padding: const EdgeInsets.symmetric(horizontal: 4.0),
-                        child: DiagnosticsNodeDescription(node.diagnostic)),
-                  ),
-                ),
-              ),
-            ],
+            children: children,
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+
+class ConstraintsDescriptor extends StatefulWidget {
```

Comment:
maybe move this widget to its own file.
debug_layout.dart 
?

Reply:
will go ahead and do that then :D